components/LiteMintNFT.vue

```diff
@@ -71,9 +71,10 @@ const { wallet, signer } = storeToRefs(store)
 const { initIfNecessary } = store
 
 const error = ref('')
-const isLoading = ref(false)
+const _isLoading = ref(false)
 
-const iscnData = ref<any>(null)
+const localISCNData = ref<any>(null)
+const localISCNId = ref('')
```

Comment:
I think `props.iscnData` doesn't clash with `iscnData` as a ref so this rename is optional